adaptation_layer/driver/fivegr_so.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, nfvo_cred):
     self._port = nfvo_cred["port"] if "port" in nfvo_cred else 8080
     self._headers = {"Content-Type": "application/json",
                      "Accept": "application/json"}
-
+    print("[DEBUG] _host:{} _port:{}".format(self._host, self._port))
```

Comment:
Can you please use the `logging` module instead of `print`?

You should add this at the beginning of your driver:
```
import logging
logger = logging.getLogger('app.driver.fivergr_so')
```

Then, your debug line should become:

```
logger.debug("_host:{} _port:{}".format(self._host, self._port))
```
Thanks.